lib/v2/screens/profile_screens/profile/profile_screen.dart

```diff
@@ -27,14 +30,40 @@ class ProfileScreen extends StatelessWidget {
               case PageState.failure:
                 return const FullPageErrorIndicator();
               case PageState.success:
-                return ListView(
-                  children: <Widget>[
-                    const ProfileHeader(),
-                    const DividerJungle(thickness: 2),
-                    const ProfileMiddle(),
-                    const DividerJungle(thickness: 2),
-                    const ProfileBottom(),
-                  ],
+                return Scaffold(
+                  appBar: AppBar(
+                    title: BlocBuilder<ProfileBloc, ProfileState>(builder: (context, state) {
+                      return DropdownButton<bool>(
+                        value: true,
+                        icon: const Icon(Icons.keyboard_arrow_down_sharp, color: AppColors.white),
+                        underline: const SizedBox.shrink(),
+                        hint: Text(state.profile?.account ?? '', style: Theme.of(context).textTheme.headline6),
```

Comment:
can we make the profile and the account not nullable? 

The user cant load this page unless its logged in, so the profile must be there.